zebra-test/src/command.rs

```diff
@@ -847,6 +848,92 @@ impl<T> TestChild<T> {
         }
     }
 
+    /// Checks each line of the child's stdout, until it finds every regex in `success_regexes`,
+    /// and returns all the lines matched by any regex. Prints all stdout lines.
```

Comment:
If I understand the method correctly, it would be more accurate to say that it returns the first matching line for each regex:

```suggestion
    /// and returns the first matching line for each regex or a list of unmatched regexes.
    /// Prints all stdout lines.
```

Reply:
It's not quite "the first matching line for each regex", it's "all lines matched by any regex until each regex has matched" :

```suggestion
    /// and returns all lines matched by any regex, until each regex has been matched at least once.
    /// If the output finishes or the command times out before all regexes are matched, returns an error with
    /// a list of unmatched regexes. Prints all stdout lines.
```